### PR TITLE
osd/atxxxx move to new WQ and uORB::Subscription

### PR DESCRIPTION
**Pull request by dagar**
@DanielePettenuzzo can you quickly test this one? The init is a bit different.

### CODE REVIEW

**Comment by DanielePettenuzzo** [comment]
Cool. I have a setup with an omnibus so I can test it tomorrow or early next week.

**Comment by dagar** [comment]
Could someone with the hardware give this a quick sanity test? We need to get everything moved into our own WQs. https://github.com/PX4/Firmware/issues/12068

File: src/drivers/osd/atxxxx/atxxxx.cpp [hunk 1]
```diff
@@ -200,52 +173,46 @@ OSDatxxxx::init_osd()
...
 
 int
 OSDatxxxx::add_character_to_screen(char c, uint8_t pos_x, uint8_t pos_y)
 {
-
 	uint16_t position = (OSD_CHARS_PER_ROW * pos_y) + pos_x;
-	uint8_t position_lsb;
-	int ret;
+	uint8_t position_lsb = 0;
+	int ret = PX4_ERROR;
```
Comment (bkueng) [inline src/drivers/osd/atxxxx/atxxxx.cpp]: This change is unnecessary.

Comment (dagar) [reply src/drivers/osd/atxxxx/atxxxx.cpp]: I know, but I'd rather initialize everything as a general rule.

**Pull request merged**